src/js/child_process.js

```diff
@@ -2,10 +2,18 @@
 
 var Pipe = require('pipe_wrap').Pipe;
 var EventEmitter = require('events').EventEmitter;
-var StringDecoder = require('string_decoder').StringDecoder;
 var util = require('util');
 var net = require('net');
-var SIGNAL_NO = require('constants').os.signals;
+var constants = require('constants');
+var SIGNAL_NO = constants.os.signals;
+var INTERNAL_IPC_HEADER_LENGTH_SIZE = constants.INTERNAL_IPC_HEADER_LENGTH_SIZE;
+var INTERNAL_IPC_HEADER_SIZE = constants.INTERNAL_IPC_HEADER_SIZE;
+var INTERNAL_IPC_PAYLOAD_MAX_SIZE = constants.INTERNAL_IPC_PAYLOAD_MAX_SIZE;
```

Comment:
Define all above constants only in child_process.js if those won't be used by others?